src/gpodder/youtube.py

```diff
@@ -155,13 +155,15 @@ def find_urls(page):
                 error_message = util.remove_html_tags(x['reason'][0])
             elif 'player_response' in x:
                 player_response = json.loads(x['player_response'][0])
-
-                if 'reason' in player_response['playabilityStatus']:
-                    error_message = util.remove_html_tags(player_response['playabilityStatus']['reason'])
-                elif 'live_playback' in x:
+                playabilityStatus = player_response['playabilityStatus']
+
+                if 'reason' in playabilityStatus:
+                    error_message = util.remove_html_tags(playabilityStatus['reason'])
+                elif 'liveStreamability' in playabilityStatus (
+                    and not playabilityStatus['liveStreamability'].get('liveStreamabilityRenderer', {}).get('displayEndscreen', False)):
```

Comment:
```
                elif 'liveStreamability' in playabilityStatus \
                        and not playabilityStatus['liveStreamability'].get('liveStreamabilityRenderer', {}).get('displayEndscreen'):
```
use antislash instead of parenthesis and indent once more ;-)

Reply:
Thanks, that worked.